### ISSUE TITLE
Auto-config.lisp emit prefixed symbols

### ISSUE THREAD

**Issue by jmercouris** [atlas-engineer/nyxt#1156]
When setting the configuration for a class within a package that is not `NYXT` for slots that are also not within the package `NYXT`, auto-config.lisp should automatically generate package prefixed symbols.

For example, it should generate:

```
(define-configuration nyxt/web-mode:web-mode
   ...)
```

rather than:
```
(define-configuration web-mode
   ...)
```

**Comment by jmercouris** [comment]
We now do this, but the problem is we only have 'configure' buttons for slots that have 'user' in them. The opposite behavior should be in effect. Otherwise a form such as:

```
Appending to "/home/jmercouris/.config/nyxt/auto-config.lisp":
(DEFINE-CONFIGURATION NYXT/WEB-MODE:USER-WEB-MODE
  ((GLYPH "7")))
```

which is incorrect, we actually want:

```
Appending to "/home/jmercouris/.config/nyxt/auto-config.lisp":
(DEFINE-CONFIGURATION NYXT/WEB-MODE:WEB-MODE
  ((GLYPH "7")))
```

**Comment by Ambrevar** [comment]
Should be fixed with #2194.

**Comment by Ambrevar** [comment]
No longer relevant since #2194 is on master.
